lib/modules/datasource/util.ts

```diff
@@ -8,3 +11,30 @@ export function isArtifactoryServer<T = unknown>(
 ): boolean {
   return is.string(res?.headers[JFROG_ARTIFACTORY_RES_HEADER]);
 }
+
+export async function getGoogleAuthToken(): Promise<string | null> {
+  try {
+    const googleAuth: GoogleAuth = new GoogleAuth({
+      scopes: 'https://www.googleapis.com/auth/cloud-platform',
+    });
+    const accessToken = await googleAuth.getAccessToken();
+    if (accessToken) {
+      // sanitize token
+      addSecretForSanitizing(accessToken);
+      return Buffer.from(`${'oauth2accesstoken'}:${accessToken}`).toString(
```

Comment:
```suggestion
      return Buffer.from(`oauth2accesstoken:${accessToken}`).toString(
```

Template literals are (still) unnecessary for static strings - not sure why you didn't use the suggested snippet :grimacing: